openapi/ga4gh-tool-discovery.yaml

```diff
@@ -643,6 +650,11 @@ definitions:
       updated:
         type: string
         description: Last time the container was updated.
+      hashcode:
+        type: string
+        description: Helpful for immutable tool versions, but can be independent. This exposes the hashcode for specific image versions to verify that the container version pulled is actually the version that was indexed by the registry.
```

Comment:
Are you saying that the hashcode can be used for things other than immutable tool versions? The phrase `can be independent does` sounds a little off to me.

Reply:
Yeah, I think it could potentially be useful even if the version is not immutable (i.e. if I want to check whether the version I got is what I expected to get, even if the version might be expected to change in the future).